src/components/login-box.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import './style.css';
+
+import { TextLabel } from './text-label';
+import { TextInput } from './text-input';
+import { ButtonInput } from './button-input';
+
+import { validateEmail, validatePassword } from '../helpers/login-validation';
+
+export const LoginBox = () => {
+  const [email, setEmail] = React.useState('');
+  const [password, setPassword] = React.useState('');
+  const [emailError, setEmailError] = React.useState('');
+  const [passwordError, setPasswordError] = React.useState('');
+
+  const onChangeEmail = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setEmail(event.target.value);
+  };
+
+  const onChangePassword = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setPassword(event.target.value);
+  };
+
+  const onSubmit = () => {
+    setEmailError(validateEmail(email));
+    setPasswordError(validatePassword(password));
+    if (emailError === '' && passwordError === '') {
+      console.log(`Email: ${email}`);
+      console.log(`Password: ${password}`);
+    } else {
+      console.log(emailError + ' ' + passwordError);
+    }
+  };
+
+  return (
+    <div className='LoginBox'>
+      <div className='LoginBox__content'>
+        <TextLabel type='header'>Bem-vindo(a) à Taqtile!</TextLabel>
```

Comment:
O conceito de "label" está mais para um texto que faz uma indicação de outro componente visual na tela. Indicação de um campo de texto, por exemplo (como num formulário). Aqui, me parece que a função desse texto na página é de título mesmo, correto? Nesse caso, talvez seja melhor usarmos a nomenclatura padrão H1, H2, H3, ou até mesmo algo como `Title`, caso vc queira.

Reply:
EU estava buscando centralizar todo componente de texto em um só arquivo, por isso criei o `TextLabel`, mas posso renomear para `Text` somente, que tal?

Reply:
Acho que ok. A componentização agora no início pode ficar a seu critério.